core/modules/qdisp/JobStatus.h

```diff
@@ -37,15 +37,21 @@ namespace lsst {
 namespace qserv {
 namespace qdisp {
 
-/// ExecStatus instances receive timestamped reports of execution State. This
-/// allows a manager object to receive updates on status without exposing its
-/// existence to a delegate class. The ExecStatus class could be extended to
-/// save all received reports to provide a timeline of state changes, but this
-/// is not currently implemented.
-class ExecStatus {
```

Comment:
I renamed refnum to jobId, Spec to JobDescription and ExecStatus to JobStatus, I think this make more sense because all of this entities are related to a Job (which is what run on the worker for 1 chunk query). I'll do it in separate ticket in the future. Please note that this refactoring allowed me to better understand the code ans was very useful to complete the ticket.